__tests__/app.test.js

```diff
@@ -117,3 +117,40 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200: should respond with comments associated with the article_id sorted by date", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments.length > 0).toBe(true);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+        });
+      });
+  });
+  test("GET 404: should respond with appropriate status and msg if requesting non-existent id", () => {
+    return request(app)
+      .get("/api/articles/99999/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("No comments available");
+      });
```

Comment:
This test is valid but it's for when the article doesn't exist so the error message should be related to that. You are missing 1 test. What if the article does exist but has no comments associated with it? For example article id 13? In this case, it should respond with the empty array, not an error as we need a place for any future comments to be stored 🙂 Please do add this test